src/modules/user-interactions.js

```diff
@@ -0,0 +1,120 @@
+export const INVOLVEMENT_API = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
+
+export const appId = async () => {
+  const response = await fetch(`${INVOLVEMENT_API}/apps/`,
+    {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(),
+    });
+  const id = await response.text();
+  const key = localStorage.getItem('football');
+  if (key === null) {
+    localStorage.setItem('football', id);
+    const newKey = localStorage.getItem('football');
+    return newKey;
+  }
+  return key;
+};
+
+export const addComments = async (id, data) => {
+  const response = await fetch(`${INVOLVEMENT_API}/apps/${id}/comments/`,
+    {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    });
+
+  const user = await response.json().then((data) => data).catch((err) => err);
+  return user;
+};
+
+export const fetchComments = async (id, itemId) => {
+  const response = await fetch(`${INVOLVEMENT_API}/apps/${id}/comments/?item_id=${itemId}`,
+    {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+  const comments = await response.json().then((data) => data).catch((err) => err);
+  return comments;
+};
+
+export const displayComments = (list) => {
+  let str = '';
+  if (list === null) {
+    str = '<li class="flex">No Comment</li>';
+  } else {
+    list.forEach((data) => {
+      str += ` <li class="flex">
+      <p>${data.creation_date}</p>
+      <p><b>${data.username}:</b></p>
+      <p>${data.comment}</p>
+    </li>`;
+    });
+    document.querySelector('.comments').innerHTML = str;
+  }
+};
```

Comment:
- [ ] Good job working on this, but the comments on the modal popup does not get fetched when the modal opens not until a new comment is made. Please kindly ensure that the comments get loaded and displayed when the modal is popped up